pkg/testutil/recorder.go

```diff
@@ -92,12 +92,13 @@ func (r *recorderStream) Record(a Action) {
 }
 
 func (r *recorderStream) Action() (acts []Action) {
+out:
```

Comment:
I also think the original code more clearer, but go vet doesn't like it :(

Should we just keep the original code?